src/lib/haversineFunction.js

```diff
@@ -0,0 +1,17 @@
+export default function getDistanceFromLatLonInKm(lat1, lon1, lat2, lon2) {
+  function deg2rad(deg) {
+    return deg * (Math.PI / 180);
+  }
+  var R = 6371; // Radius of the earth in km
```

Comment:
The earth isn’t perfectly round, so an improvement here would be to adjust the radius based on the user’s latitude. Not **at all** saying you should do this now! 😅